src/controllers/users.controller.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const {
+  getUsers,
+  createUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/users.serveces');
+
+const getAllUsers = (req, res) => {
+  res.statusCode(200);
+  res.send(getUsers());
+};
+
+const createOneUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = createUser(name);
+
+  res.statusCode(201);
+
+  res.send(user);
+};
+
+const getOneUser = (req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    res.sendStatus(400);
```

Comment:
You should add return after `res.sendStatus(400)` to stop the execution of the function if id is not provided.